src/authentication/DPoPWebIdExtractor.ts

```diff
@@ -13,31 +13,38 @@ import { CredentialsExtractor } from './CredentialsExtractor';
 export class DPoPWebIdExtractor extends CredentialsExtractor {
   protected readonly logger = getLoggerFor(this);
   private readonly targetExtractor: TargetExtractor;
+  private readonly solidIdentityVerifier: VerifySolidIdentityFunction;
 
   public constructor(targetExtractor: TargetExtractor) {
     super();
     this.targetExtractor = targetExtractor;
+    this.solidIdentityVerifier = createSolidIdentityVerifier();
   }
 
   public async canHandle({ headers }: HttpRequest): Promise<void> {
-    const { authorization } = headers;
-    if (!authorization || !authorization.startsWith('DPoP ')) {
-      throw new NotImplementedHttpError('No DPoP Authorization header specified.');
+    const { authorization, dpop } = headers;
+    if (!authorization) {
+      throw new BadRequestHttpError('No authorization header specified.');
+    }
+    if (!authorization.startsWith('DPoP ')) {
+      throw new BadRequestHttpError('Authorization header is not DPoP bound.');
+    }
+    if (!dpop) {
+      throw new BadRequestHttpError('No DPoP header specified.');
     }
   }
 
   public async handle(request: HttpRequest): Promise<Credentials> {
     const { headers: { authorization, dpop }, method } = request;
-    if (!dpop) {
-      throw new BadRequestHttpError('No DPoP token specified.');
-    }
     const resource = await this.targetExtractor.handleSafe(request);
+
     try {
-      const webId = await verify(authorization as string, dpop as string, method as any, resource.path);
-      this.logger.info(`Verified WebID via DPoP token: ${webId}`);
-      return { webId };
+      // eslint-disable-next-line max-len
+      const { webid } = await this.solidIdentityVerifier(authorization as string, dpop as string, method as RequestMethod, resource.path);
```

Comment:
Perhaps 
```suggestion
      const { webid: webId } = await this.solidIdentityVerifier(authorization as string, dpop as string, method as RequestMethod, resource.path);
```

already here for consistency with our variable names?